expression/column.go

```diff
@@ -376,17 +376,22 @@ func IndexInfo2Cols(cols []*Column, index *model.IndexInfo) ([]*Column, []int) {
 	return retCols, lengths
 }
 
-// FindColumnsByUniqueIDs will find columns by checking the unique id.
-// Note: `ids` must be a subset of the column slice.
-func FindColumnsByUniqueIDs(cols []*Column, ids []int) []*Column {
+// FindColumnsByUniqueIDList will find columns by checking the unique id.
+// Then return order is the same with `ids`. If one id doesn't exists in `cols`. We'll just break.
+func FindColumnsByUniqueIDList(cols []*Column, ids []int64) []*Column {
```

Comment:
how about:
- `s/FindColumnsByUniqueIDList/FindPrefixOfIndexCols/`
- `s/ids/indexColIDs`

Reply:
I think `OfGivenCols` is more better?

Reply:
But it seems that only when the second parameter is column ids of an index the prefix is returned. In other situation, all the matched columns should be returned?

By the way, the caller of this function now only pass the column ids of an index, except for the test code.